File: 2652-sum-multiples/2652-sum-multiples.java
class Solution {
    public int sumOfMultiples(int n) {
        int res = 0;
        for(int i=0; i<=n;i++)
            if(i%3==0||i%5==0||i%7==0)
                res+=i;
        return res;
    }
}

// Time Complexity: O(n)
// Space Complexity: O(1)

File: 2652-sum-multiples/2652-sum-multiples.py
class Solution:
    def sumOfMultiples(self, n: int) -> int:
        res = 0
        for i in range(1,n+1):
            if i%3 == 0 or i%5 ==0 or i%7 == 0:
                res+=i
        return res
# Time Complexity: O(n)
# Space Complexity: O(1)
        